profiler/snippets/snippets.go

```diff
@@ -19,16 +19,26 @@ package main
 
 import (
 	"cloud.google.com/go/profiler"
+	"google.golang.org/api/option"
 )
 
 func main() {
-	// Profiler initialization, best done as early as possible.
-	if err := profiler.Start(profiler.Config{
+	cfg := profiler.Config{
 		Service:        "myservice",
 		ServiceVersion: "1.0.0",
 		// ProjectID must be set if not running on GCP.
 		// ProjectID: "my-project",
-	}); err != nil {
+	}
+
+	// For OpenCensus users that use Cloud Trace and Cloud Monitoring:
+	// cloud.google.com/go/profiler creates its own spans when making its
+	// own requests to send profiling data to the server, which can be
+	// noisy. This option prevents the profiler package from adding spans
+	// for its own requests.
+	opts := []option.ClientOption{option.WithTelemetryDisabled()}
```

Comment:
This needs to be updated to use the flag, and the snippet should show how to turn on the telemetry since default is to turn it off.

Reply:
Fixed to use EnableOCTelemetry flag.